src/AuthorizedSocketConnection.js

```diff
@@ -34,49 +22,34 @@ type MaybeSubscription = Promise<
   AsyncIterator<ExecutionResult> | ExecutionResult,
 >;
 
-type ServerConfig = {|
+type ServerConfig<TContext, TCredentials> = {|
```

Comment:
i think this should be `SocketConfig` or `SocketOptions` or `AuthorizedSocketOptions` or something